src/main/api/browser.ts

```diff
@@ -0,0 +1,96 @@
+import { ok } from "assert";
```

Comment:
Please use `import { ok } from "readium-desktop/common/utils/assert";` (remember, we found critical runtime bugs with the `assert` module!)

Reply:
done